configs/acra-keys.yaml

```diff
@@ -1,7 +1,4 @@
 version: 0.85.0
-# client ID for which to retrieve key
-client_id: 
```

Comment:
has this moved somewhere or we just got rid of it?

Reply:
> has this moved somewhere or we just got rid of it?

It's still there in a sense that you can still, for example, add `keys_dir` to the configuration file so that you don't have to remember to add it to each individual subcommand. However, the handling of `--dump_config` currently takes into account only the global options. All interesting options have moved to subcommands, so as a result the template for configuration file became a bit barren.

I'm still thinking about what can be done for this. We can't just update the template manually because the tests verify that `--dump_config` outputs exactly this config. Making `acra-keys` an exception is not pretty. We need some way to compose options from multiple subcommands, deduplicate them, output all options into a configuration file, and make sure that – when used in the config – options of one subcommand do not break other subcommands.

Reply:
> We need some way to compose options from multiple subcommands

if this is complicated / long / unclear -> goto2; break;

goto2: make `acra-keys` an exception.

Reply:
Well, I have an idea for this, but it will require some additional refactoring in `--dump_config` handling, to make it possible to construct a configuration from multiple flag sets. I'd like to make it in a different PR and settle for an incomplete config for now.

Reply:
awesome, thank you!